rules/do-you-know-how-to-claim-expense-reimbursements-2/rule.md

```diff
@@ -50,54 +40,32 @@ At SSW we find the greatest overall value for expense reimbursements is to use t
  **Employees:** 
 
 
-
-
-  1. Using the Xero Expense mobile app, submit a photo of the receipt. If the receipt was sent via email, you can still submit a photo or screenshot of the email receipt
-
-
-
-
-
-
-
+1. Open the Xero Expense mobile app and submit a photo of the receipt for reimbursement or filing. If the receipt was sent via email, you can still submit a photo or screenshot of the email receipt
 
 ![](2020-11-25_15-44-09.png)
 
-
  **Figure: Xero Expense app submit a photo** 
 
 
 
-
-
 2. Either allow the app to analyse the details manually or enter the details yourself (if you are strapped for time, you can take a photo and come back to it or hold on to the receipt and attend to it when possible)
 
 
-
-
 ![](e.jpg)
 
 
  **Figure: Details required of a submitted receipt** 
 
- **
-** 
-
-3. Whether you allow the app to analyse the details or not, you will still be required to confirm whether the expense was paid personally or with company money
-
-
 
+3. Confirm whether the expense was paid personally or with company money
 
 
 ![](2020-11-25_15-51-14.png) 
 
  **Figure: Option of reimbursable and non-reimbursable** 
 
 
-
-4. Submit for approval and then wait to be reimbursed
-
-
+4. When all details have been checked and are complete, click submit for approval and then wait to be reimbursed. If you are not ready to submit and run out of time, you can save as draft and get back to it by the next day
```

Comment:
"If you are not ready to submit and run out of time, you can save it as a draft and come back to it later."

I think this wording is better, what do you think?

Reply:
Yes, agreed